src/web/tests/helpers/ValidateTask.tsx

```diff
@@ -0,0 +1,58 @@
+import { MissionTask, TaskType } from "../../jcc/client/components/shared/JAIAProtobuf";
+//Helper function to verify the task created by the Panel is valid
+export function validateTask(task?: MissionTask): void {
+    if (!task) {
+        return;
+    }
+    switch (task?.type) {
+        case TaskType.CONSTANT_HEADING:
+            //make sure all the parameters are defined
```

Comment:
Nitpicking: Should we capitalize and add a space with the comment? Not sure just want it to be consistent.